README.md

```diff
@@ -76,11 +74,13 @@ version 9.2 or earlier will _NOT_ work.
 Operating System | Perl
 ---------------- | ----
 CentOS 7         | 5.16                        
-Debian 7         | 5.14
-FreeBSD 10.3     | 5.24
-FreeBSD 11.1     | 5.24
+Debian 8         | TBD
+Debian 9         | TBD
```

Comment:
Perl => 5.24

Reply:
@sandoche2k, do you know what version of Perl that comes with Debian 8 and Debian 9?

Reply:
Debian 8 => 5.20
Debian 9 => 5.24